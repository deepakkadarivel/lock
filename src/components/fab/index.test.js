import React from 'react';
import { create } from 'react-test-renderer';
import Fab from './index';

describe('Fab component', () => {
    test('Renders Fab component', () => {
        const component = create(<Fab onClick={() => {}} icon="add" />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
