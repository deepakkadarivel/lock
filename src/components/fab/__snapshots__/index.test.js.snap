// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fab component Renders Fab component 1`] = `
<button
  className="Fab ripple"
  onClick={[Function]}
  type="button"
>
  <i
    className="material-icons"
  >
    add
  </i>
</button>
`;
